### PR TITLE
treewide merge: major directory clean up

### PR DESCRIPTION
**Pull request by leroymusa**

### CODE REVIEW

**Review by leroymusa — COMMENTED** [review]
reviewed and satisfied

**Review by acrfactory — COMMENTED** [review]
GENIUS

File: isolation tests/altimeter_test/BMP280/BMP280.ino [hunk 1]
```diff
@@ -0,0 +1,42 @@
...
+  
+
+  for(int i=0; i < 100; i++){
+    t1 = millis();
+    bmp.readTemperature();
+    
+    t2 = millis();
+    t_int = t2 - t1;
+    Serial.println(bmp.readPressure());
+    //Serial.println(t_int);
```
Comment (Joas329) [inline isolation tests/altimeter_test/BMP280/BMP280.ino:36]: Dont add unnecessary comments

File: isolation tests/altimeter_test/BMP280/BMP280.ino [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  Serial.println("Adafruit BMP280 test");
+
+
+    if (!bmp.begin(0x77)) { /* Failed to initialize BMP280*/ while (1); } // Halt the program
+     bmp.setSampling(Adafruit_BMP280::MODE_NORMAL,     /* Operating Mode. */
+                  Adafruit_BMP280::SAMPLING_X2,     /* Temp. oversampling */
+                  Adafruit_BMP280::SAMPLING_X16,    /* Pressure oversampling */
+                  Adafruit_BMP280::FILTER_X16,      /* Filtering. */
+                  Adafruit_BMP280::STANDBY_MS_125); /* Standby time. */
+
```
[the remark this answers is not in the document]
Comment (leroymusa) [reply isolation tests/altimeter_test/BMP280/BMP280.ino:22]: bruh

**Pull request merged**